scripts/lynx/authorize_enrico.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
```

Comment:
What do you think above moving this script to the common set of scripts `scripts/` and use click options? We're probably going to have to do something similar for `tapir` / `mainnet`.

Could do something like
```
ape run authorize_enrico --network polygon:mumbai:infura --registry-filepath <> --enrico <>
```

Reply:
sounds like a great idea 👍🏻 we're gonna need documentation 😅

Reply:
opened https://github.com/nucypher/nucypher-contracts/issues/154

Reply:
✔️